monitoring-bot/internal/monitor.go

```diff
@@ -109,6 +135,19 @@ func NewMonitor(envPath string, jsonPath string) (Monitor, error) {
 
 	return mon, nil
 }
+func validateMnemonic(mnemonic string, network string) error {
```

Comment:
while this removes out the issue of having symbols, it doesn't enforce any rules on the number of words. https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/grid-client/deployer/tf_plugin_client.go#L116